components/navbar/navbar.snip.html

```diff
@@ -22,12 +22,17 @@
     <a role="button" on="tap:{{sidebar-id}}.toggle"class="ampstart-navbar-trigger {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} pr2">☰</a>
     {{#logo}}
       {{#href}}
-      <a href="{{.}}" class="text-decoration-none inline-block">
+      <a href="{{.}}" class="text-decoration-none inline-block mx-auto">
       {{/href}}
-      <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="fixed" class="my0 mx-auto {{classes}}" alt="{{name}}"></amp-img>
+      {{#url}}
+        <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="fixed" class="my0 mx-auto {{classes}}" alt="{{name}}"></amp-img>
+      {{/url}}
+      {{^url}}
+        <h1 class="h3 bold caps mx-auto {{classes}}">{{name}}</h1>
```

Comment:
It’s quite common for site names to be inside an `<h1>` but I don’t object to changing it to `<div>` or `<p>` if you prefer (it’s already inside a `<header>` element).